src/main/java/com/tabnine/inline/render/GraphicsUtils.kt

```diff
@@ -21,6 +21,11 @@ object GraphicsUtils {
         return Font(attributes)
     }
 
+    val color: Color
+        get() {
+            return Color(AppSettingsState.instance.color)
+        }
```

Comment:
Not sure I understand why this needs to reside here. I don't like a god util classes, and this seems like one.